packaging/change_name.py

```diff
@@ -0,0 +1,53 @@
+#!/usr/bin/env python3
+"""
+Barebones utility for changing the name of the package in `pyproject.toml`.
+
+Usage
+-----
+[SCRIPT] [PYPROJECT_FILE] [NAME]
+
+Notes
+-----
+Should only be used by the CI
```

Comment:
This isn't helpful to because I don't know if it's telling me that it'll only work during CI; or it's just a hint that it's mostly used during CI. This can be folded into the first sentence.

Reply:
Removed the comment because we can use the script wherever we want, it's just that it's kinda pointless to use it outside of the CI.